src/aequitas/plotting.py

```diff
@@ -0,0 +1,1064 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+import matplotlib.gridspec as gridspec
+import re
+import seaborn as sns
+import math
+from aequitas.squarify_flipped import normalize_sizes, squarify, padded_squarify, \
+    squarify_plot_rects
+import warnings
+
+# Authors: Pedro Saleiro <saleiro@uchicago.edu>
+#          Loren Hinkson
+#
+# License: Copyright \xa9 2018. The University of Chicago. All Rights Reserved.
+
+class Plotting(object):
+    """
+    """
+
+    def __init__(self, key_metrics=None, key_disparities=None):
+        """
+        :param key_metrics: Set default absolute group metrics for all subplots
+        :param key_metrics: Set default disparity metrics for all subplots
+        :param fill_divbyzero:
+        """
+        if not key_metrics:
+            self.key_metrics = ['pprev', 'ppr', 'fdr', 'for', 'fpr', 'fnr']
+        else:
+            self.key_metrics = key_metrics
+        if not key_disparities:
+            self.key_disparities = \
+                ['pprev_disparity', 'ppr_disparity', 'fdr_disparity',
+                     'for_disparity', 'fpr_disparity', 'fnr_disparity']
+        else:
+            self.key_disparities = key_disparities
```

Comment:
not a big deal but this can be made more transparent, and the code more straightforward, by putting the defaults in the signature, _e.g._:

    all_key_metrics = ('pprev', 'ppr', 'fdr', 'for', 'fpr', 'fnr')
    all_key_disparities = ('pprev_disparity', 'ppr_disparity', 'fdr_disparity', 'for_disparity', 'fpr_disparity', 'fnr_disparity')

    def __init__(self, key_metrics=all_key_metrics, key_disparities=all_key_disparities):
        self.key_metrics = list(key_metrics)
        self.key_disparities = list(key_disparities)

(here I've assumed that it's really important that these attributes be of type `list`.)